packages/pass-style/src/types.d.ts

```diff
@@ -188,7 +191,10 @@ export type RemotableMethodName = PropertyKey;
 /**
  * The authority-bearing leaves of a Passable's pass-by-copy superstructure.
  */
-export type PassableCap = Promise<any> | RemotableObject;
+export type PassableCap =
+  | Promise<any>
+  | RemotableObject
+  | RemotableBrand<any, any>;
```

Comment:
Ok, I see you do make a similar elaboration of `PassableCap`. Same comments apply regarding meaning.

For both before and after, why is this distinct from `PassByRef`?